src/Physics/Multinucleon/EventGen/MECGenerator.cxx

```diff
@@ -1333,3 +1303,59 @@ void MECGenerator::LoadConfig(void)
     GetParamDef( "SuSA-MaxXSec-DiffTolerance", fSuSAMaxXSecDiffTolerance, 999999. );
 }
 //___________________________________________________________________________
+double MECGenerator::GetNSVXSecMaxTlctl( const Interaction * in, 
+			   const Range1D_t Tl_range, 
+			   const Range1D_t ctl_range ) const {
+
+  ROOT::Math::Minimizer * min = ROOT::Math::Factory::CreateMinimizer("Minuit2");
+
+  gsl::d2Xsec_dTCosth f(fXSecModel,in) ; 
```

Comment:
If we don't change the gsl function, here you can de-reference the interaction calling
```C++
  gsl::d2Xsec_dTCosth f(fXSecModel, & in) ; 
```